pipelines/test/tw/ldd-check.yaml

```diff
@@ -22,7 +22,12 @@ inputs:
 pipeline:
   - name: "check for missing library dependencies using ldd"
     runs: |
+      if [ -z "${{inputs.files}}" ] && [ -z "${{inputs.packages}}" ]; then
+          packages="${{context.name}}"
```

Comment:
It seems like `context.name` isn't defined:

```
Error: compiling build: compiling subpackage "acl-dev" tests: compiling Pipeline[1]: compiling Pipeline[0]: mutating runs: variable context.name not defined
```

(This is from the regular pipeline build)

Reply:
Dann just told me that `context.name` is new.  Marking this discussion as resolved.